src/js/components/molecules/cards/card-article/card-article.vue.html

```diff
@@ -9,4 +9,5 @@ <h3 class="card-article__heading">{{ title }}</h3>
     <p v-if="description" class="card-article__description">
         {{ description }}
     </p>
+    <list-unordered class="card-article__list" :listItems="listItems"/>
```

Comment:
So yeah, mabye just a bunch of `<span>`s in a `<p>`?

Reply:
can't figure out how to get typescript to agree.